  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50

  joint1_position_controller:
    type: effort_controllers/JointPositionController
    joint: manipulator_base_joint
    pid: {p: 30.0, i: 10.0, d: 0.0}

  joint2_position_controller:
    type: effort_controllers/JointPositionController
    joint: manipulator_joint_1
    pid: {p: 100.0, i: 15.0, d: 10.0}